Submitted By: Ken Moffat <ken@linuxfromscratch.org>
Date: 2005-09-09
Initial Package Version: 2.10.0
Upstream Status: unknown
Origin: from gentoo
Description: Allows gpdf to compile on x86_64 with gcc4
(apply on top of the gcc4 patch)

--- gpdf-2.10.0/xpdf/gpdf-control.cc	2005-05-20 15:03:29.000000000 +0200
+++ gpdf-2.10.0.az/xpdf/gpdf-control.cc	2005-05-20 15:06:39.000000000 +0200
@@ -209,12 +209,12 @@
 			   const gchar * status)
 {
 	GPdfControlPrivate *priv;
-	guint to_id;
+	unsigned long to_id;
 
 	g_return_if_fail (GPDF_IS_NON_NULL_CONTROL (control));
 	priv = control->priv;
 
-	if ((to_id = (unsigned int)
+	if ((to_id = (unsigned long)
 	     g_object_get_data (G_OBJECT (control),
 				"status-timeout-id")) != 0) {
 		g_object_set_data (G_OBJECT (control),
